Actions/Deliver/Deliver.ps1

```diff
@@ -78,69 +78,70 @@ try {
         }
-        elseif ($artifacts -eq "current" -or $artifacts -eq "prerelease" -or $artifacts -eq "draft") {
-            # latest released version
-            $releases = GetReleases -token $token -api_url $ENV:GITHUB_API_URL -repository $ENV:GITHUB_REPOSITORY
-            if ($artifacts -eq "current") {
-                $release = $releases | Where-Object { -not ($_.prerelease -or $_.draft) } | Select-Object -First 1
-            }
-            elseif ($artifacts -eq "prerelease") {
-                $release = $releases | Where-Object { -not ($_.draft) } | Select-Object -First 1
-            }
-            elseif ($artifacts -eq "draft") {
-                $release = $releases | Select-Object -First 1
-            }
-            if (!($release)) {
-                throw "Unable to locate $artifacts release"
-            }
-            New-Item $artifactsFolder -ItemType Directory | Out-Null
-            $artifactsFolderCreated = $true
-            $artifactFile = DownloadRelease -token $token -projects $project -api_url $ENV:GITHUB_API_URL -repository $ENV:GITHUB_REPOSITORY -release $release -path $artifactsFolder -mask "Apps"
-            Write-Host "'$artifactFile'"
-            if (!$artifactFile -or !(Test-Path $artifactFile)) {
-                throw "Artifact $artifacts was not found on any release. Make sure that the artifact files exist and files are not corrupted."
-            }
-            if ($artifactFile -notlike '*.zip') {
-                throw "Downloaded artifact is not a .zip file"
-            }
-            Expand-Archive -Path $artifactFile -DestinationPath ($artifactFile.SubString(0,$artifactFile.Length-4))
-            Remove-Item $artifactFile -Force
-        }
         else {
-            New-Item $artifactsFolder -ItemType Directory | Out-Null
-            $artifactsFolderCreated = $true
-            $atypes.Split(',') | ForEach-Object {
-                $atype = $_
-                $allArtifacts = GetArtifacts -token $token -api_url $ENV:GITHUB_API_URL -repository $ENV:GITHUB_REPOSITORY -mask $atype -projects $project -version $artifacts -branch $refname
-                if ($allArtifacts) {
-                    $allArtifacts | ForEach-Object {
-                        $artifactFile = DownloadArtifact -token $token -artifact $_ -path $artifactsFolder
-                        Write-Host $artifactFile
-                        if (!(Test-Path $artifactFile)) {
-                            throw "Unable to download artifact $($_.name)"
-                        }
-                        if ($artifactFile -notlike '*.zip') {
-                            throw "Downloaded artifact is not a .zip file"
-                        }
-                        Expand-Archive -Path $artifactFile -DestinationPath ($artifactFile.SubString(0,$artifactFile.Length-4))
-                        Remove-Item $artifactFile -Force
-                    }
+            $artifactsFolder = Join-Path $baseFolder ".artifacts"
```

Comment:
If you don't show whitespace changes - the change is really not that big of a change.
It is just ensuring that artifactsFolder is created at all times and my test repository for testing this PR ran into this - so moving this out, would mean that this would fail until I had that merged (or modify my test repo of course)

Reply:
All the more reason to open a separate PR :) If it's a small change, we can review it and merge it relatively quickly. Right now, the fix won't go in until we've reviewed the fix + the new troubleshooting action. 

Hiding the whitespaces did help though! Thanks for the tip 👍